### DISCUSSION TITLE
How to download PF v14 ?

### DISCUSSION THREAD

**Discussion by vitsin** [primefaces/community#1819] [PrimeFaces] [answered]
hello,

1.How to download PF v14 ?
2.I have Elite subscription and it gives to download PF 12.0.7 - max, no PF v14 available at all.

regards,
--vs

P.S.
PrimeFaces now become free? I can download it from Maven repository ...

**Comment by melloware** [comment, marked as answer]
Yes its on Maven central.

```xml
<dependency>
    <groupId>org.primefaces</groupId>
    <artifactId>primefaces</artifactId>
    <version>14.0.0</version>
</dependency>
```
